articles/search/search-agentic-retrieval-how-to-create.md

```diff
@@ -236,7 +236,6 @@ Content-Type: application/json
             "indexName" : "{{index-name}}",
             "filterAddOn" : "State eq 'WA'",
             "IncludeReferenceSourceData": true, 
-            "rerankerThreshold " : 2.5,
```

Comment:
The problem with line 239 is the extra space in rerankerThreshold.  Can you remove it?  I can accept your PR if you make the modification.  Thank you!